package.json

```diff
@@ -267,7 +267,9 @@
       "@fluentui/web-components/ts-loader/**",
       "@fluentui/web-components/ts-node",
       "@fluentui/web-components/ts-node/**",
-      "@fluentui/web-components/webpack"
+      "@fluentui/web-components/webpack",
+      "@storybook/addons/@storybook/router/react-router",
```

Comment:
`@storybook/router` uses `react-router`+`react-router-dom` version 6 (which include their own types). Northstar docs and e2e use version 5 + `@types/react-router-dom`. Without this line, yarn was hoisting the version 6 deps, and those were being used to resolve the `react-router` types in northstar, causing compiler errors.